tfhe/src/shortint/engine/mod.rs

```diff
@@ -92,6 +92,13 @@ where
     assert_eq!(accumulator.polynomial_size(), polynomial_size);
     assert_eq!(accumulator.glwe_size(), glwe_size);
 
```

Comment:
This function could take an ShortintEncoding as input
I don't know this is a better though